index.html

```diff
@@ -6,276 +6,307 @@
+              <li class="tag">JS</li>
+              <li class="tag">CSS</li>
+              <li class="tag">Ruby & Rails</li>
+            </ul>
+            <button class="btn-popup" type="button" title="see-project">See Project</button>
+          </article>
+
+          <article class="card">
+            <div class="card-image">
+              <img class="img-plaholders" src="assets/images/Img-plaholder.svg" alt="Img-plaholder" />
             </div>
-          </li>
+            <h2 class="project-title">Multi-Post Stories Gain+Glory</h2>
+            <ul class="tags">
+              <li class="tag">HTML</li>
+              <li class="tag">JS</li>
+              <li class="tag">CSS</li>
+              <li class="tag">Ruby & Rails</li>
+            </ul>
+            <button class="btn-popup" type="button" title="see-project">See Project</button>
+          </article>
         </ul>
       </section>
-      <section class="about-me">
-        <div class="text-on-about">
-          <h2 class="primary-text">About me</h2>
+      <!-- about me -->
+      <section class="container-flex about-myself" id="about-myself-id">
+        <div class="container">
+          <h1 class="primary-text">About me</h1>
           <p class="supporting-text-about">
-            I can help you build a product , <br />
-            feature or website Look through <br />
-            some of my work and experience! <br />
-            If you like what you see and have <br />
-            a project you need coded, <br />
-            don’t hesitate to contact me.
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
           </p>
-          <button class="butn" type="button" title="resume">Get my resume</button>
+          <button class="button" type="button" title="resume">
+            Get my resume
+          </button>
         </div>
-        <ul class="card-about">
+
+        <ul class="card-about-about-myself">
           <li class="card-works-inactive">
-            <img src="assets/images/language-icons.svg" alt="languages" />
-            <h3 class="title">Languages</h3>
-            <ul class="frame19-tags">
-              <li class="tag">HTML</li>
-              <li class="tag">JS</li>
-              <li class="tag">CSS</li>
-              <li class="tag">Ruby</li>
-            </ul>
+            <article class="card-categories">
+              <div class="categorie-image">
+                <img src="assets/images/icon-languages.svg" alt="languages" />
+              </div>
+              <h2 class="categorie-title">Languages</h2>
+              <ul class="frame19-tags">
+                <li class="tag">HTML</li>
+                <li class="tag">JS</li>
+                <li class="tag">CSS</li>
+                <li class="tag">Ruby</li>
+              </ul>
+            </article>
           </li>
```

Comment:
![image](https://github.com/yesuf023/Portfolio/assets/60083437/9bfda426-161e-4550-b41a-7e46175f66fc)

- [ ] You have used `color: lightgray` as the background of `HTML`, `JS`,`CSS`,`Ruby & Rails`. But as per the given design guideline the `background color` should be `#EBF0EE`. Kindly fix this issue.

![image](https://github.com/yesuf023/Portfolio/assets/60083437/d3d5f3a0-f7b7-478d-8b3e-40ef38b1fff4)